examples/messenger/terra/src/lib.rs

```diff
@@ -117,18 +117,14 @@ mod testing {
                 funds:         vec![],
                 msg:           Binary::from(&[
                     123, 34, 112, 111, 115, 116, 95, 109, 101, 115, 115, 97, 103, 101, 34, 58, 123,
-                    34, 109, 101, 115, 115, 97, 103, 101, 34, 58, 34, 87, 122, 77, 115, 77, 67,
-                    119, 119, 76, 68, 65, 115, 78, 106, 89, 115, 77, 84, 69, 120, 76, 68, 107, 52,
-                    76, 68, 69, 120, 76, 68, 65, 115, 77, 67, 119, 119, 76, 68, 99, 121, 76, 68,
-                    69, 119, 77, 83, 119, 120, 77, 68, 103, 115, 77, 84, 65, 52, 76, 68, 69, 120,
-                    77, 83, 119, 122, 77, 105, 119, 50, 78, 83, 119, 120, 77, 68, 103, 115, 77, 84,
-                    65, 49, 76, 68, 107, 53, 76, 68, 69, 119, 77, 86, 48, 61, 34, 44, 34, 110, 111,
-                    110, 99, 101, 34, 58, 48, 125, 125
+                    34, 109, 101, 115, 115, 97, 103, 101, 34, 58, 34, 65, 119, 65, 65, 65, 69,
+                    74, 118, 89, 103, 115, 65, 65, 65, 66, 73, 90, 87, 120, 115, 98, 121, 66, 66, 98,
+                    71, 108, 106, 90, 81, 61, 61, 34, 44, 34, 110, 111, 110, 99, 101, 34, 58, 48, 125, 125
                 ]),
             }))]
         );
     }
-
+/*
```

Comment:
This test fails due to the fact that cosmwasm doesn't support mocking out cross-contract calls currently. Rather than commenting it out, for now can we remove the `#[test]` attribute and leave the function intact instead of committing commented code, and then adding a comment above this with something like:

```
/// This test currently would fail until cosmwasm can mock cross contract calls, see:
/// https://github.com/CosmWasm/cosmwasm/blob/0.16/packages/std/src/mock.rs#L389
```